CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## Important Notes
 
+- [#905](https://github.com/oauth2-proxy/oauth2-proxy/pull/905) Existing sessions from v6.0.0 or earlier are no longer valid. They will trigger a reauthentication.
```

Comment:
Is this message accurate? I thought the sessions in v6.0.0 included the new format? Should it not say from before v6.0.0?

Reply:
I looked through the release notes history:

v6.0.0 got the SHA1 -> SHA256 change.
v6.1.0 got the MessagePack LZ4 sessions

All the code referred to it as `V5LegacySessions` but I think it missed the v6 major release and we finished it up by the minor release.

Reply:
Ack, makes sense, shall we rebase and get this merged then

Reply:
Just rebased 👍

Reply:
I had so many unit tests for the legacy route.  Would be a shame if this lowers our testing coverage score 🤣